test/onigumo_cli_test.exs

```diff
@@ -83,6 +83,16 @@ defmodule OnigumoCLITest do
       end
     end
 
+    test("Usage message ends with exactly one newline") do
+      output = capture_io(fn -> Onigumo.CLI.main([]) end)
+      assert String.match?(output, ~r/\N\n\z/)
+    end
+
+    test("Help message ends with exactly one newline") do
```

Comment:
Maybe I’d group the tests separately for the usage message and the help message. That means, move this above `help_message_printed?`.